### DISCUSSION TITLE
关于清爽首页排行榜只显示top3

### DISCUSSION THREAD

**Discussion by GlowsSama** [the1812/Bilibili-Evolved#3611] [功能建议 / Ideas]
不知道排行版能否扩展下  像V1是可以上下翻页
而V2目前想看排行榜就必须点击“排行榜”跳转到二级页面
感觉有些反逻辑  交互上也麻烦了许多

**Comment by the1812** [comment]
虽然之前拒过一次 (#3175), 不过现在又想到个办法能多显示点了
